crates/biome_js_semantic/src/events.rs

```diff
@@ -493,6 +493,7 @@ impl SemanticEventExtractor {
             self.push_binding(None, BindingName::Value(name), info);
             false
         };
+        self.binding_count += 1;
```

Comment:
I think this operation shouldn't be here, but it should be inside `push_binding`

Reply:
It cannot, because we can push two binding names with the same binding index. Maybe I should rename `binding_count` to `declaration_count` and/or `push_binding` to `push_binding_name`...